src/Logging/OmexLoggingOptions.cs

```diff
@@ -23,5 +23,10 @@ internal class OmexLoggingOptions
 		/// Maximum number of events that activity can store for replay
 		/// </summary>
 		public uint MaxReplayedEventsPerActivity { get; set; } = 1000;
+
+		/// <summary>
+		/// Enable or disable OmexLogger
+		/// </summary>
+		public bool OmexLoggerEnabled { get; set; } = false;
```

Comment:
Why isn't it just "Enabled"?
It is already under OmexLoggingOptions
```suggestion
		public bool Enabled { get; set; } = false;
```

Reply:
I prefer OmexLogger here as to make it explicit

Reply:
It is already explicit. It is OmexLoggingOptions.
OmexLoggingOptions only apply to OmexLogger...